### DISCUSSION TITLE
How to link GuestOrder to Logged in User ActiveOrder

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1542] [Q&A]
In Storefront user added items to the cart before authentication. During the checkout process, the user will log in. at this point how to add the order that is there in the cart to the user's active order?

**Comment by michaelbromley** [comment]
This should automatically occur - the contents of the guest order will be transferred over to a new order seamlessly. However, if the customer while last logged in already had some items in the cart, then the contents of the guest order will get merged. The exact behaviour of this is defined by the [OrderMergeStrategy](https://www.vendure.io/docs/typescript-api/orders/order-merge-strategy/)

**Reply by ramkishore-yerubandi** [reply]
The authentication Strategy I am using is Bearer token-based and cookies are disabled. In this case, also the merge would happen?

**Reply by michaelbromley** [reply]
Yes, that should not make a difference.

**Reply by ramkishore-yerubandi** [reply]
ok. Thanks!
